MAUI/Cartesian-Charts/neumorphicUI-column-chart.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: Neumorphic UI Column Chart in .NET MAUI Chart control | Syncfusion
```

Comment:
@DhanarajRajendran 
Modify the title as Neumorphic UI Chart in .NET MAUI Chart control